scripts/steamed_hams.py

```diff
@@ -173,55 +194,82 @@ def steamed_hams_parser():
     sys.exit(-1)
 
 # Configure python first
-modules = {"brian": "",
-           "experiment_handler": "",
-           "radar_control": "",
-           "data_write": "",
-           "realtime": "",
-           "rx_signal_processing": "",
-           "usrp_driver": ""}
+modules = {
+    "brian": "",
+    "experiment_handler": "",
+    "radar_control": "",
+    "data_write": "",
+    "realtime": "",
+    "rx_signal_processing": "",
+    "usrp_driver": "",
+}
 
 for mod in modules.keys():
     opts = python_opts.format(module=mod)
-    modules[mod] = f"source borealis_env{PYTHON_VERSION}/bin/activate; python{PYTHON_VERSION} {opts} src/{mod}.py" \
-
-modules['data_write'] = modules['data_write'] + " " + data_write_args
-modules['usrp_driver'] = modules['usrp_driver'] + " " + f'{mode} --c_debug_opts="{c_debug_opts}"'
-modules['experiment_handler'] = modules['experiment_handler'] + " " + args.experiment_module + " " + \
-                                args.scheduling_mode_type
+    modules[mod] = (
+        f"source borealis_env{PYTHON_VERSION}/bin/activate; python{PYTHON_VERSION} {opts} src/{mod}.py"
+    )
+modules["data_write"] = modules["data_write"] + " " + data_write_args
+modules["usrp_driver"] = (
+    modules["usrp_driver"] + " " + f'{mode} --c_debug_opts="{c_debug_opts}"'
+)
+modules["experiment_handler"] = (
+    modules["experiment_handler"]
+    + " "
+    + args.experiment_module
+    + " "
+    + args.scheduling_mode_type
```

Comment:
Another wonky format - its consistent which I like, just in some cases it seems to take up more lines than needed